planning/behavior_path_planner/autoware_behavior_path_planner_common/src/utils/path_utils.cpp

```diff
@@ -471,34 +471,36 @@ std::vector<Pose> interpolatePose(
 {
   std::vector<Pose> interpolated_poses{};  // output
 
-  const std::vector<double> base_s{
-    0, autoware::universe_utils::calcDistance2d(start_pose.position, end_pose.position)};
-  const std::vector<double> base_x{start_pose.position.x, end_pose.position.x};
-  const std::vector<double> base_y{start_pose.position.y, end_pose.position.y};
-  std::vector<double> new_s;
-
-  constexpr double eps = 0.3;  // prevent overlapping
-  for (double s = eps; s < base_s.back() - eps; s += resample_interval) {
-    new_s.push_back(s);
-  }
+  const double distance =
+    autoware::universe_utils::calcDistance2d(start_pose.position, end_pose.position)
+};
```

Comment:
This code does not look correct to me :thinking: Maybe an extra `}` was introduced by mistake ?

Reply:
Thank you for informing me!
That must be a mistake, I'll fix it soon

Reply:
@maxime-clem 
I have fixed the problem. I am sorry to bother you, but could you review it again?